### ISSUE TITLE
Laravel 11 - doctrine/dbla req

### ISSUE THREAD

**Issue by wit3** [tpetry/laravel-postgresql-enhanced#79]
Hi,
I'm tring to install the package in a new project created with L11 ( sail + pgsql + redis )
But the composer need doctrine/dbal that is removed in L11, any plan to replace that request or suggest ?

I'm wrong with any step?  the new clean install it's a basic install 😖

**Comment by tpetry** [comment]
There are no plans to remove the `doctrine/dbal` dependency currently as that would break support for any Laravel installation below 11. And I don't want to maintain two different versions.

**Comment by GringoDotDev** [comment]
Hi @tpetry! Pardon the ping on a closed issue.

To get around the composer installation error, I'm passing the --with-all-dependencies flag.

This works, but it looks like it involves downgrading a framework package:

```
- Downgrading carbonphp/carbon-doctrine-types (3.2.0 => 2.1.0)
```

Just wanted to verify that this is correct / the recommended way to do it?

**Comment by tpetry** [comment]
Hey Gringo. The former request was about dropping Doctrine because Laravel 11 doesn‘t need it anymore. Can you share when and where the downgrade is happening? Because as Laravel 11 just removed the dependency and my package still requires it, there won‘t be any difference in the installed packages.

**Comment by tpetry** [comment]
@GringoDotDev The issue about needing to downgrade has been fixed with version `0.40.0` 😉

**Comment by GringoDotDev** [comment]
I totally forgot to follow up on this, thank you so much for the update!!
